test/Makefile

```diff
@@ -33,6 +33,7 @@ TESTS=\
       format_unittest \
       health_checker_unittest \
       resource_unittest \
+      json_unittest \
```

Comment:
Want to keep these in alphabetical order?

Reply:
Done.